### ISSUE TITLE
Changing profiles crashes 205

### ISSUE THREAD

**Issue by ogrotten** [TanninOne/modorganizer#684]
Pic:  http://imgur.com/kikeoRA
Crash log sent referencing this issue number.

The source profile is named "default" and is my main game-playing profile. The destination profile is named "Blank" and has no mods selected. 

1. Installed MO, using the installer, to a new directory. It appeared to have caught my previous info from 204 because first run ran without the tutorial screens and it didn't ask me anything. 
1. I wanted to switch profiles to **Blank ** for the first run of the game to make sure it just ran.
1. MO Crashed while switching profiles.

**Comment by ogrotten** [comment]
I changed the directories used by 205 to something other than what 204 used, so I could remove version differences from the equation. I then copied the downloads and mods over to the new dir.

The only profile was Default. I created a Blank profile. When switching to the new profile, MO crashed in the same way.

If I do not change profiles, but go straight to the game, the game runs fine.

**Comment by WhiteZeroX** [comment]
Same thing here.

Heres my logs:
[MOCrash.zip](https://github.com/TanninOne/modorganizer/files/258354/MOCrash.zip)

**Comment by rdunlap** [comment]
same here. 
[ModOrganizer.exe.dmp.zip](https://github.com/TanninOne/modorganizer/files/258453/ModOrganizer.exe.dmp.zip)

**Comment by neokio** [comment]
Could be related to #702.

**Comment by sbroadbent** [comment]
I'd like to add my name to the list of people also getting a crash (or rather MO2.05 becomes unresponsive possibly due to the quantiy of mods) when I switch profiles.  I don't have any log files, and can't figure out how to enable logging.

I messed around with paths, which messed things up, and when I got it fixed, none of my mods in the left hand pane were selected.  At that point I tried switching profiles, and it worked, and switched profiles without crashing.

I re-enabled a number of mods, but got a crash.

I also randomly enabled a couple mods, in this case the following mods (Ammunition Re-texture, Armor Prefix Fix, Assault Gas Mask and Green Hood - Recolors, Base Power Armor Skins, Better Quarry Sites), and got no crash.  None of these have an esp/esm.

After a few random tests, as long as there are no more than one esp/esm in the right hand pane, MO2.05 will change profiles without crashing.  If there is more than one esp/esm managed by MO on the right side pane (which for most of us will be a definite yes), it crashes.

I am finally getting MO to generate a log file:

19:18:10.236 [I] create config in 5368
19:18:10.236 [I] 0 processes
19:18:10.236 [I] mod_organizer_instance_1 created in process 5368
19:18:10.236 [I] attached to mod_organizer_instance_1 with 1 nodes, size 65536
19:18:10.236 [I] inv_mod_organizer_instance_1 created in process 5368
19:18:10.236 [I] attached to inv_mod_organizer_instance_1 with 1 nodes, size 65536
